src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -788,19 +788,20 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         return builder->CreateCall(fn, {str});
     }
 
-    llvm::Value* lfortran_str_copy(llvm::Value* str, llvm::Value* idx1, llvm::Value* idx2)
+    llvm::Value* lfortran_str_copy(llvm::Value* str, llvm::Value* idx1, llvm::Value* idx2, llvm::Value* idx3 = nullptr)
```

Comment:
I think instead `idx3` we should use `step` as the name.

Reply:
`idx3` was used just for consistency with `idx1` for left and `idx2`for right. I've updated the name now.